examples/bri-3/src/bri/shared/constants.ts

```diff
@@ -0,0 +1,2 @@
+// DTO validation messages
```

Comment:
not a blocker, just a suggestion -> how about using `common` instead of `shared`

Reply:
For me it is completely the same. Not sure what others think.